packages/react-native-mmkv/src/useMMKVDevTools.ts

```diff
@@ -1,104 +1,105 @@
 import { useDevToolsPluginClient, type EventSubscription } from 'expo/devtools';
-import { useCallback, useEffect } from 'react';
-import { MMKV } from 'react-native-mmkv';
+import { useCallback, useEffect, useMemo } from 'react';
+import { createMMKV, type MMKV } from 'react-native-mmkv';
 
 import { Method } from '../methods';
 
-/**
- * This hook registers a devtools plugin for react-native-mmkv.
- *
- * The plugin provides you with the ability to view, add, edit, and remove react-native-mmkv entries.
- *
- * @param props
- * @param props.errorHandler - A function that will be called with any errors that occur while communicating
- * with the devtools, if not provided errors will be ignored. Setting this is highly recommended.
- * @param props.storage - A MMKV storage instance to use, if not provided the default storage will be used.
- */
-export function useMMKVDevTools({
-  errorHandler,
-  storage = new MMKV(),
-}: {
+type Params = {
   errorHandler?: (error: Error) => void;
   storage?: MMKV;
```

Comment:
```suggestion
  storage: MMKV;
```

let's make `storage` required. Then we don't need to worry about how to construct mmkv and the plugin potentially stays compatible with both v3 and v4 (and so on)